gridstatus/pjm.py

```diff
@@ -80,7 +80,333 @@ class PJM(ISOBase):
         "124076095",
         "970242670",
         "1709725933",
-    ]
+        '5021703',]
```

Comment:
looks like there is one extra here. is this supposed to be under `price_node_ids`